From a4efb86d144d4214143f6d79372795c5a5f4ad5c Mon Sep 17 00:00:00 2001
From: Tatsuyuki Ishi <ishitatsuyuki@gmail.com>
Date: Wed, 17 Aug 2016 14:49:24 +0900
Subject: [PATCH] Always assume pthread working

---
 TSRM/threads.m4 | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/TSRM/threads.m4 b/TSRM/threads.m4
index 38494ce..25882b4 100644
--- a/TSRM/threads.m4
+++ b/TSRM/threads.m4
@@ -85,12 +85,8 @@ int main() {
   ], [
   pthreads_working=no
   ], [
-  dnl For cross compiling running this test is of no use. NetWare supports pthreads
-  pthreads_working=no
-  case $host_alias in
-  *netware*)
-    pthreads_working=yes
-  esac
+  dnl We have already checked for pthread in buildroot config
+  pthreads_working=yes
 ]
 ) ] )dnl
 dnl
-- 
2.8.1

